config/nephe.yml

```diff
@@ -510,6 +522,20 @@ rules:
   - get
 ---
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: secret-viewer-role
```

Comment:
Its not a good practice to allow nephe-controller to read all the secrets in all namespace.. We should ask user to put the secrets in nephe-namespace only